packages/@okta/eslint-plugin-okta-ui/lib/rules/no-missing-keys.js

```diff
@@ -0,0 +1,41 @@
+const loginEnBundle = require('../../../i18n/src/json/login.json');
+const loginokPLBundle = require('../../../i18n/src/json/login_ok_PL.json');
+const _ =  require('underscore');
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Detect missing keys from login_ok_PL.properties',
+      category: 'i18n Issues',
+      recommended: true,
+    },
+    messages: {
+      noMissingKeys: 'login.properties ( {{noOfKeysLoginEn}}: keys ) and login_ok_PL.properties ( {{noOfKeysLoginokPL}}: keys), are out of sync. Missing keys from login_ok_PL.properties. Run grunt exec:pseudo-loc to re-generate login_ok_PL.properties.'
+    },
+  },
+  create(context) {
+    return {
+      'Program': function (node) {
+        const fileName = context.getFilename();
+        if (fileName.includes('login.json')) {
+          const noOfKeysLoginEn = Object.keys(loginEnBundle).length;
+          const noOfKeysLoginokPL = Object.keys(loginokPLBundle).length;
+          // check if the en and ok_PL bundles are in sync
+          const bundlesInSync = _.isEqual( Object.keys(loginEnBundle),Object.keys(loginokPLBundle));
+          if (!bundlesInSync) {
+            let loc = node.loc.start;
+            context.report({
+              messageId: 'noMissingKeys',
+              node,
+              data: {
+                noOfKeysLoginEn,
+                noOfKeysLoginokPL
```

Comment:
I'm not sure this information is valuable to developers, as any number of keys reported here will result in the same remediation. Given that, I think we can omit it.

Reply:
Yeah, I was just trying to give them an indication of how many keys are missing so when they run the pseudo-loc command that many keys should get added to that file.

Reply:
removed to keep the message generic